bundle.Dockerfile

```diff
@@ -1,17 +1,13 @@
 FROM scratch
 
-LABEL com.redhat.openshift.versions="v4.8"
-LABEL com.redhat.delivery.backport=true
-LABEL com.redhat.delivery.operator.bundle=true
-
 # Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=mongodb-atlas-kubernetes
-LABEL operators.operatorframework.io.bundle.channels.v1=stable
-LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.34.1
+LABEL operators.operatorframework.io.bundle.channels.v1=beta
+LABEL operators.operatorframework.io.bundle.channel.default.v1=beta
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.35.0
```

Comment:
this looks strange. What is it beta and 1.35?

Reply:
Reverted back to stable